bedrock/settings/static_media.py

```diff
@@ -1464,7 +1436,7 @@
         ),
         'output_filename': 'js/growth-firstrun-test2-bundle.js',
     },
-    'home-2015': {
+    'home': {
         'source_filenames': (
             'js/base/mozilla-share-cta.js',
```

Comment:
Hm, I don't see our `share_cta` macro on the home page either. Maybe remove this file also?